tests/tracking/test_tracking.py

```diff
@@ -8,11 +8,20 @@
 import mlflow
 from mlflow import tracking
 from mlflow.entities import RunStatus
+from mlflow.exceptions import MlflowException
 from mlflow.tracking.fluent import start_run, end_run
 from mlflow.utils.file_utils import TempDir
+from mlflow.utils.mlflow_tags import MLFLOW_PARENT_RUN_ID
 
 
-def test_create_experiment():
+@pytest.fixture()
+def temporary_tracking_uri(tmpdir):
```

Comment:
Nit: we can use the existing tracking URI mock in `from tests.projects.utils import tracking_uri_mock`

Reply:
Nice, let's do that.